src/components/BookImage.tsx

```diff
@@ -0,0 +1,23 @@
+import { Image } from '@nextui-org/react'
+import noImage from '/no-image.png'
+
+export default function BookImage({
+  image = '',
+  title = '',
+  customClasses = '',
+}: {
+  image?: string
+  title?: string
+  customClasses?: string
+}) {
+  return (
+    <Image
+      src={image || noImage}
+      alt={title}
+      fallbackSrc={noImage}
+      radius="none"
+      className={`bg-cover ${customClasses}`}
+      removeWrapper
+    />
+  )
+}
```

Comment:
[SUG] We could make it a little more general to take advantage of the image component props, something like this

```suggestion
import { Image, ImageProps } from '@nextui-org/react'
import noImage from '/no-image.png'

interface BookImageProps extends ImageProps {
  image?: string
  title?: string
  customClasses?: string
}

export default function BookImage({
  image = '',
  title = '',
  customClasses = '',
  ...props
}: BookImageProps) {
  return (
    <Image
      src={image || noImage}
      alt={title}
      fallbackSrc={noImage}
      radius="none"
      className={`bg-cover ${customClasses}`}
      removeWrapper
      {...props}
```

Reply:
I like it!